claim/README.md

```diff
@@ -73,7 +73,7 @@ When coming from  [Nodes view page](https://learn.netdata.cloud/docs/cloud/visua
 If you want to connect a node that is running on a Linux environment, the script that will be provided to you by Netdata Cloud is the [kickstart](/packaging/installer/README.md#automatic-one-line-installation-script) which will install the Netdata Agent on your node, if it isn't already installed, and connect the node to Netdata Cloud. It should be similar to:
 
 ```
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
+wget -O ./kickstart.sh https://my-netdata.io/kickstart.sh && sh ./kickstart.sh --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
```

Comment:
"https://app.netdata.cloud" is the [default](https://github.com/netdata/netdata/blob/162f2b62b2dc6e38e8cd41c63bd36998b1c180fd/packaging/installer/kickstart.sh#L24), so `--claim-url` is not really needed.

Reply:
Good to know. I've included it as the other examples all had it. 
Is it even possible to have a custom claim url at the moment?
@ilyam8

Reply:
Not really (if we don't count us testing not in prod env). 

>  I've included it as the other examples all had it.

I see, ok.